<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScQtNeuron_AP_Window</class>
 <widget class="QMainWindow" name="ScQtNeuron_AP_Window">
  <property name="geometry">
   <rect>
    <x>400</x>
    <y>600</y>
    <width>548</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot drawing routine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../3rdParty/qcustomplot/include/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
